packages/frontend/src/components/ChartDownload.tsx

```diff
@@ -210,6 +211,7 @@ interface ChartDownloadMenuProps {
 
 export const ChartDownloadMenu: React.FC<ChartDownloadMenuProps> = memo(
     ({ getCsvLink }) => {
+        const { projectUuid } = useParams<{ projectUuid: string }>();
```

Comment:
can we have `projectUuid` as a prop instead of using `useParams`?
ideally params should only be used in `Page` components

my experience: I was refactoring settings pages (project settings) which included changing routes. even tho project compiled and tests passed 2 components (deeply nested, one of them was modal) were broken because they used `useParams`

reusable components should be as isolated from side effects as possible

Reply:
Oh great catch! Will do 👍